gym/lib/gym/generators/build_command_generator.rb

```diff
@@ -43,7 +43,7 @@ def options
           options << "-scmProvider system"
         end
         options << config[:xcargs] if config[:xcargs]
-        options << "OTHER_SWIFT_FLAGS=\"-Xfrontend -debug-time-function-bodies\"" if config[:analyze_build_time]
+        options << "OTHER_SWIFT_FLAGS=\"\\\$(value) -Xfrontend -debug-time-function-bodies\"" if config[:analyze_build_time]
```

Comment:
I'm not too familiar with this but is `$(value)` the same thing as `$(inherited)`? 🙃

Reply:
I don't know where or if there is official documentation for it, but `$(value)` expands to the current build setting value. So it's sort of the command-line equivalent of `$(inherited)`

Reply:
Ohhhhhh, cool cool! I see what this is doing now 🤯 Thank you for the explanation 🥰